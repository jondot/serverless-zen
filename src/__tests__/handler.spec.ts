import { get } from './fixtures'

const { hello } = require('../handler')

describe('handler#hello', () => {
  it('should say hello', async () => {
    const res = await hello(...get())
    expect(res).toMatchSnapshot()
  })
})
