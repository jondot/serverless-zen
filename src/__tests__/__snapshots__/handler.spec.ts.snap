// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handler#hello should say hello 1`] = `
Object {
  "body": "{\\"hello\\":\\"world\\"}",
  "isBase64Encoded": false,
  "multiValueHeaders": Object {
    "content-type": Array [
      "application/json",
    ],
  },
  "statusCode": 200,
}
`;
